app/code/core/Mage/Adminhtml/Block/Report/Grid/Abstract.php

```diff
@@ -85,7 +85,7 @@ protected function _getAggregatedColumns()
 
     /**
      * Add column to grid
-     * Overriden to add support for visibility_filter column option
+     * Override to add support for visibility_filter column option
```

Comment:
I think overridden was correct in this comment

Reply:
It depends on how you relate to verb tenses. There is a present and a past action of the same verb.

Reply:
In this case the comment explains that the function was "overridden in order to add support for etc etc", if i'm reading the code correctly

Reply:
@fballiano this was found with phpstorms spellcheck  ... lets keep it ond move on.

Reply:
@fballiano  either it describes what this metthod

- does (present) or
- did (past)

Im no native english speaker, but "Override" makes more sense for explain the purpose of it.

Reply:
that's why I took time to read the method and explain it here. and I commented because to me, after reading the code, it make more sense before. spell check doesn't have the context.
anyway.

Reply:
Me too. :yum: 

As said weird to think about it ... (independent from outcome)

... finally we saved two chars :P

(one for change, one for wrong spelling)